### DISCUSSION TITLE
[i18n] How to translate title from configuration

### DISCUSSION THREAD

**Discussion by mariuszkrzaczkowski** [facebook/docusaurus#8609] [Q&A]
does not translate title added from configuration in title tag in head
```
title: 'YetiForce Documentation',
```
https://github.com/YetiForceCompany/YetiForceDoc/blob/64ef2b81eded60c3430d513237da99375370dba0/docusaurus.config.js#L8

it only translates the first part and not the second
```html
<title>Wprowadzenie | YetiForce Documentation</title>
```
https://doc.yetiforce.com/pl/introduction

**Comment by slorber** [comment]
There are some things that we, unfortunately, can't translate for now.

Please report them here if not already mentioned: https://github.com/facebook/docusaurus/issues/4542

(I think this one is already reported, website title is not translatable)
